projects/1.16/assets/forbidden-arcanus/forbidden_arcanus/lang/zh_cn.json

```
@@ -0,0 +1,361 @@
+    "effect.forbidden_arcanus.fly": "飞行",
+    "effect.forbidden_arcanus.spectral_vision": "幽灵视觉",
+    "effect.forbidden_arcanus.darkened": "漆黑",
+
+    "category_blocks": "方块",
+    "block.forbidden_arcanus.darkstone": "暗黑石",
+    "block.forbidden_arcanus.darkstone_slab": "暗黑石砖",
+    "block.forbidden_arcanus.darkstone_stairs": "暗黑石台阶",
+    "block.forbidden_arcanus.darkstone_wall": "暗黑石墙",
+    "block.forbidden_arcanus.arcane_gilded_darkstone": "奥术镀金黑石",
+    "block.forbidden_arcanus.polished_darkstone": "磨制暗黑石",
+    "block.forbidden_arcanus.polished_darkstone_slab": "磨制暗黑石台阶",
+    "block.forbidden_arcanus.polished_darkstone_stairs": "磨制暗黑石楼梯",
+    "block.forbidden_arcanus.polished_darkstone_wall": "磨制暗黑石墙",
+    "block.forbidden_arcanus.polished_darkstone_pressure_plate": "磨制暗黑石压力板",
+    "block.forbidden_arcanus.polished_darkstone_button": "磨制暗黑石按钮",
+    "block.forbidden_arcanus.chiseled_polished_darkstone": "錾制磨制暗黑石",
+    "block.forbidden_arcanus.runic_chiseled_polished_darkstone": "錾制磨制符文暗黑石",
+    "block.forbidden_arcanus.arcane_chiseled_polished_darkstone": "神秘錾制磨制暗黑石",
+    "block.forbidden_arcanus.polished_darkstone_bricks": "磨制暗黑石砖",
+    "block.forbidden_arcanus.polished_darkstone_brick_slab": "磨制暗黑石砖台阶",
+    "block.forbidden_arcanus.polished_darkstone_brick_stairs": "磨制暗黑石砖楼梯",
+    "block.forbidden_arcanus.polished_darkstone_brick_wall": "磨制暗黑石砖墙",
+    "block.forbidden_arcanus.cracked_polished_darkstone_bricks": "裂纹磨制暗黑石砖",
+    "block.forbidden_arcanus.arcane_polished_darkstone": "神秘磨制暗黑石",
+    "block.forbidden_arcanus.arcane_polished_darkstone_slab": "神秘磨制暗黑石台阶",
+    "block.forbidden_arcanus.arcane_polished_darkstone_stairs": "神秘磨制暗黑石楼梯",
+    "block.forbidden_arcanus.arcane_polished_darkstone_wall": "神秘磨制暗黑石墙",
+    "block.forbidden_arcanus.chiseled_arcane_polished_darkstone": "錾制神秘磨制暗黑石",
+    "block.forbidden_arcanus.arcane_polished_darkstone_pillar": "神秘磨制暗黑石柱",
+    "block.forbidden_arcanus.arcane_polished_darkstone_rod": "神秘磨制暗黑石烛",
+
+    "block.forbidden_arcanus.dark_beacon": "暗黑信标",
+    "block.forbidden_arcanus.arcane_glass": "神秘玻璃",
+    "block.forbidden_arcanus.arcane_glass_pane": "神秘玻璃板",
+    "block.forbidden_arcanus.runic_tenebris_frame": "暗黑符文框架",
+    "block.forbidden_arcanus.runic_tenebris_core": "暗黑符文核心",
+    "block.forbidden_arcanus.dark_runestone": "暗黑符石",
+    "block.forbidden_arcanus.arcane_dark_stone": "神秘暗黑石",
+    "block.forbidden_arcanus.runic_glass": "符文玻璃",
+    "block.forbidden_arcanus.runic_glass_pane": "符文玻璃板",
+    "block.forbidden_arcanus.dark_runic_glass": "暗黑符文玻璃",
+    "block.forbidden_arcanus.dark_runic_glass_pane": "暗黑符文玻璃板",
+    "block.forbidden_arcanus.runestone": "符石",
+    "block.forbidden_arcanus.arcane_crystal_ore": "神秘水晶矿",
+    "block.forbidden_arcanus.arcane_crystal_block": "神秘水晶方块",
+    "block.forbidden_arcanus.arcane_crystal_obelisk": "神秘水晶方尖碑",
+    "block.forbidden_arcanus.dark_nether_star_block": "暗黑下界之星方块",
+    "block.forbidden_arcanus.end_crystal_gem": "末影水晶宝石",
```

```suggestion
    "block.forbidden_arcanus.end_crystal_gem": "末地水晶宝石",
```
由原版`End Crystal`合成而来，是否应该和原版一致。

---

原版译名变动应该不涉及低版本吧

---

等改到116头上了再说